templates/rdrs-aws-TargetGroupBinding.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2024-2025 Hopsworks AB. All rights reserved.
+{{- if include "rdrs.unmanagedLoadBalancers" . -}}
+{{- if include "rondb.isAWS" . }}
+apiVersion: elbv2.k8s.aws/v1beta1
+kind: TargetGroupBinding
+metadata:
+  name: rdrs-http
+  namespace: {{ .Release.Namespace }}
+spec:
+  serviceRef:
+    name: {{ .Values.meta.rdrs.externalLoadBalancer.name }}
+    port: 4406
```

Comment:
can we template the ports instead?

Reply:
Currently it's hardcoded in all the places. I created a ticket, somebody can take it afterwards https://hopsworks.atlassian.net/browse/HWORKS-2119